src/dmd/mars.d

```diff
@@ -380,8 +380,6 @@ private int tryMain(size_t argc, const(char)** argv, ref Param params)
         import dmd.root.longdouble;
         initFPU();
     }
-    import dmd.root.ctfloat : CTFloat;
-    CTFloat.initialize();
```

Comment:
Please leave as-is. Cross-compilers need to set up a `real_t` format depending on target.

Reply:
Ok. So LDC is overwriting `CTFloat.initialize`?

Reply:
Restored the `initialize` function.